receiver/azureblobreceiver/README.md

```diff
@@ -0,0 +1,30 @@
+# Azure Blob Receiver
+
+This receiver reads logs and trace data from [Azure Blob Storage](https://azure.microsoft.com/services/storage/blobs/).
+
+## Configuration
+
+The following settings are required:
+
+- `connection_string` (no default): Azure Blob Storage connection key, which can be found in the Azure Blob Storage resource on the Azure Portal.
+- `event_hub_end_point` (no default): Azure Event Hub endpoint triggering on the `Blob Create` event 
+
+The following settings can be optionally configured:
+
+- `logs_container_name` (default = "logs"): Name of the blob container with the logs
+- `traces_container_name` (default = "traces"): Name of the blob container with the traces
```

Comment:
Do you think more options specific for logs or traces could be added in the future? If so, I would create sections for these:

```
logs:
  container_name: logs
traces:
  container_name: traces
```